assets/scss/components/_auto-complete.scss

```diff
@@ -29,6 +30,11 @@ Markup:
   </label>
 </fieldset>
 
+ComponentData:
+<script type="application/json" id="suggestions">
```

Comment:
I'd argue that we should only add JSON data, not the surrpinding `<script>` tags too. They should be added at compile time